### DISCUSSION TITLE
Project Reunion & Nuget: target framework issue

### DISCUSSION THREAD

**Discussion by LeftTwixWand** [microsoft/WindowsAppSDK#801] [Q&A] [answered]
Hi. When I'm creating a ProjectReunion solution, it has TargetFramework: net5.0-windows10.0.19041 
Why it refers to the windows build version, not just net5.0?

The problem is my local nuget server, has class library packages, that have netstandard2.1 target framework. And nuget filter think, that net5.0-windows10.0.19041 is not compatible with my nuget packages, but it is. At the same time console app with TargetFramework net5.0 sees nuget packages.

**Comment by riverar** [comment, marked as answer]
@LeftTwixWand Not seeing that behavior here, can you share more details/a sample project? Or list any errors?

[Per .NET TFM documentation](https://docs.microsoft.com/en-us/dotnet/standard/frameworks#net-5-os-specific-tfms), .NET 5.0 TFMs (OS specific or not) are compatible with .NET Standard 2.1 libraries. I was able to create a local Project Reunion app targeting `net5.0-windows10.0.19041.0` and successfully consumed a library via NuGet targeting only `netstandard2.1`.

**Reply by manodasanW** [reply]
Adding onto what @riverar said, the net5 windows version specific TFMs basically bring in the projection for the Windows WinRT API surface for the respective version to allow you to consume it in your app / library.  But if you can share more details such as which project is referencing what and what TFMs each are using, it would help understand what you are seeing.

**Reply by LeftTwixWand** [reply]
> ll

I can download nuget packages from nuget.org. But, when I'm trying to search for a package in my local nuget server - filter just ignore my request, because it's thinking, that net5.0-windows.... is not compatible with netstandard2.1

**Reply by huoyaoyuan** [reply]
It may be implementation issue of your local server. Are you on the latest version?
